octree/include/pcl/octree/octree_key.h

```diff
@@ -60,7 +60,7 @@ class OctreeKey {
   {}
 
   /** \brief Copy constructor. */
-  OctreeKey(const OctreeKey& source) { std::memcpy(key_, source.key_, sizeof(key_)); }
+  OctreeKey(const OctreeKey& source) { std::copy_n(source.key_, sizeof(key_), key_); }
```

Comment:
Why not `: key_(source.key_) {}`?

Reply:
`key_` is an array type, so it can't be initialized that way. (A long-standing beef of mine with C++...)